run-build.ps1

```diff
@@ -106,7 +106,7 @@ if ($NoBuild)
 }
 else
 {
-    dotnet msbuild build.proj /p:Architecture=$Architecture /p:GeneratePropsFile=true /t:WriteDynamicPropsToStaticPropsFiles
+    dotnet msbuild build.proj /p:Architecture=$Architecture /p:GeneratePropsFile=true /t:WriteDynamicPropsToStaticPropsFiles $ExtraParameters
```

Comment:
I think it was probably intentional that the extra parameters weren't passed to this invocation.  I'd suggest asking Livar about this.

Reply:
I don't remember why. But why do we need the extra parameters here? For the Extra restore sources? Should we make it explicit then?

Reply:
The extra parameters needed in the 'GeneratePropsFile' step are: /p:ExternalRestoreSources & /p:DotNetBuildOffline - they are needed to control the content of the NuGet.config
When reviewing the properties now available to the 'GeneratePropsFile' step; I do not believe that the behavior of this pre-step would change.